contrib/fuzz/daemon.go

```diff
@@ -2,16 +2,18 @@
 // +build gofuzz
 
 /*
-   Copyright The containerd Authors.
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-       http://www.apache.org/licenses/LICENSE-2.0
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+Copyright The containerd Authors.
```

Comment:
Other files are using space as indent. Is there any issue?

Reply:
Looks like it wanted a newline between the header comment and `package` to format it properly.
Fixed in https://github.com/containerd/containerd/pull/7254/commits/74c09b75690255ea9f42db357baf3a997228ae0a